src/main/resources/templates/jenkins/c/fact/regularRuns/pipeline.groovy

```diff
@@ -82,7 +82,7 @@ void postBuildTasks() {
     sh '''
     if [ -e test-reports/tests-results.xml ]
     then
-        sed -i 's/[^[:print:]]/�/g' test-reports/tests-results.xml
+        sed -i 's/[^[:print:]\t]/�/g' test-reports/tests-results.xml
```

Comment:
Please check or revert the \t replacement .. this is needed for stack traces (as far as I know)

Reply:
`print` does not consider it to be printable. This is a negative match group, so the tab is no longer replaced.

Reply:
E.g. message before was broken
```
�at app//org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
�at app//de.tum.in.test.api.internal.TimeoutUtils.rethrowThrowableSafe(TimeoutUtils.java:47)
�at app//de.tum.in.test.api.internal.TimeoutUtils.lambda$performTimeoutExecution$1(TimeoutUtils.java:42)
�at java.base@17.0.6/java.util.concurrent.FutureTask.run(FutureTask.java:264)
�at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
�at java.base@17.0.6/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
�at java.base@17.0.6/java.lang.Thread.run(Thread.java:833)
```
but is shown correctly now:
```
The class 'MergeSort' does not implement the interface 'SortStrategy' as expected. Implement the interface and its methods.
```

Reply:
Oh .. sorry, I did not see the "negative" matching :)